cmd/mirror-main.go

```diff
@@ -100,6 +100,10 @@ var (
 			Name:  "disable-multipart",
 			Usage: "disable multipart upload feature",
 		},
+		cli.BoolFlag{
+			Name:  "with-versioning",
+			Usage: "upload or remove with versioning enabled on a bucket",
```

Comment:
I think this doesn't clearly explain what this flag is for. Are you perhaps trying to say "enable mirroring of version history, including deletions, for versioned buckets" or "mirror objects along with their version history; versioned deletions from the source will also be mirrored to the target", and would it correctly explain what it does?